deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -36,19 +33,19 @@ rules:
     verbs: ["get", "list", "patch"]
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshotcontents"]
-    verbs: ["create", "get", "list", "watch", "update", "delete", "patch"]
+    verbs: ["get", "list", "watch", "update", "patch"]
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshotclasses"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "update", "patch"]
+    verbs: ["get", "list", "watch", "patch"]
```

Comment:
as the discussion is going on in the kubernetes-csi repo. keeping this on hold as well. https://github.com/kubernetes-csi/external-attacher/pull/354

Reply:
@Madhu-1 I have removed the update/patch for now. please revisit